### ISSUE TITLE
Error when import ThreadPool from multiprocessing.pool

### ISSUE THREAD

**Issue by Marks1** [IronLanguages/main#1551]
When 
from multiprocessing.pool import ThreadPool

i got a error 
  File "C:\Program Files (x86)\IronPython 2.7\lib\site-packages\multiprocessing\__init__.py", line 65, in <module>
  File "C:\Program Files (x86)\IronPython 2.7\lib\site-packages\multiprocessing\util.py", line 41, in <module>
ImportError: Cannot import name _args_from_interpreter_flags

**Comment by simplicbe** [comment]
Which version are you using? I think this is a known issue, let me take a look.

**Comment by simplicbe** [comment]
This bug should be fixed with #1421

**Comment by slide** [comment]
This is fixed:

```
IronPython 2.7.8a1 (2.7.0.0) on Mono 4.0.30319.42000 (64-bit)
Type "help", "copyright", "credits" or "license" for more information.
>>> from multiprocessing.pool import ThreadPool
```
